src/WebUI/Common/KubeCardWithTable.scss

```diff
@@ -1,3 +1,9 @@
 .k8s-pods-status-count {
     padding-left: 8px;
+}
+
+.external-ip-cell .external-ip-copy-icon {
+    padding: 2px;
+    vertical-align: sub;

```

Comment:
what happens if i use flex-center?

Reply:
Flex-center is already applied on these elements via parent table element. It does not align the items as needed